src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Runtimes/RuntimeDescriptor.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.Runtimes
+{
+    internal readonly struct RuntimeDescriptor
```

Comment:
I think you should be able to delete this file. This struct has only 1 string member, which you should be able to use directly without the struct.

Reply:
done. Deleted